cves/CVE-2003-0083.yml

```diff
@@ -139,7 +140,7 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: They moved to version 2.0, this was likely a very busy time. 
```

Comment:
In your previous, interesting commits, answer you said they were just starting the move to V2.0 but here you say they moved to version 2.0 (past tense). Which way is it?

Reply:
In the commits they were moving. They finished the move a few months before the vuln was released. I added a date to specify that these events are not to be taken as all happening at the same time.